helper/schema/valuetype_string.go

```diff
@@ -0,0 +1,16 @@
+// generated by stringer -type=ValueType; DO NOT EDIT
+
+package schema
+
+import "fmt"
+
+const _ValueType_name = "TypeInvalidTypeBoolTypeIntTypeStringTypeListTypeMapTypeSettypeObject"
```

Comment:
Yeah, at some point we might want to capitalize it but that use case isn't well tested to expose it as a public API and we needed it internally for private APIs.